pco_python/src/lib.rs

```diff
@@ -13,10 +13,13 @@ pub mod wrapped;
 
 pub fn core_dtype_from_str(s: &str) -> PyResult<CoreDataType> {
   match s.to_uppercase().as_str() {
+    "F16" => Ok(CoreDataType::F16),
```

Comment:
Actually, could you add these to the Python tests?

Reply:
Added 16-bit types to Python tests.